codalab/worker/docker_image_manager.py

```diff
@@ -28,6 +28,9 @@
 )
 
 
+docker_image_download_status = []
```

Comment:
Can we avoid using a global variable if possible?

Reply:
Is there any significant disadvantage? I need a variable that can be shared between threads, and the idea comes from [here](https://stackoverflow.com/questions/17774768/python-creating-a-shared-variable-between-threads) (looks like using a global variable is the most straightforward way).